transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/networkinterface/resolution/NetworkInterfaceResolutionIT.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kaazing.gateway.transport.wsn.networkinterface.resolution;
```

Comment:
This is a wrong package.  I suppose we can add tests to transport/nio module.

I also think that the test should be like TcpAcceptorIT.java (it shouldn't use any proxy service etc). Then there may some test where gateway could be started using a config file with network syntax.